external/implementations/samlconf-ddf-impl/README.md

```diff
@@ -2,7 +2,13 @@
 
 ## Steps to Test DDF's IDP
 * Start and install DDF
-* Copy the contents of `samlconf-sp-metadata.xml` to `AdminConsole -> Security -> Configuration -> IdPServer -> SP Metadata`.
+* DDF does not support having multiple `EntityDescriptor` elements inside a `EntitiesDescriptor` element. In order to work around this:
+    * Create two new xml files (doesn't matter what they're named)
```

Comment:
✏️ They don't really need to create two files. You can have them copy it directly into DDF